conans/test/integration/toolchains/cmake/test_ninja.py

```diff
@@ -78,41 +78,67 @@ def test_locally_build_linux(self):
 
     @unittest.skipIf(platform.system() != "Windows", "Only windows")
     def test_locally_build_windows(self):
-        """ Ninja build must proceed using default profile and cmake build (Windows Release)
+        """ Ninja build must proceed using default profile and cmake build (Windows)
         """
-        client = TestClient(path_with_spaces=False)
-        client.save({"conanfile.py": self.conanfile,
-                     "main.cpp": self.main_cpp,
-                     "CMakeLists.txt": self.cmake})
-        win_host = textwrap.dedent("""
-            [settings]
-            os=Windows
-            arch=x86_64
-            compiler=Visual Studio
-            compiler.version=15
-            compiler.runtime=MD
-            build_type=Release
-             """)
-        client.save({"win": win_host})
-        client.run("install . -pr=win")
-        # Ninja is single-configuration
-        vcvars = vcvars_command("15", architecture="amd64")
-        client.run_command('{} && cmake . -G "Ninja" -DCMAKE_TOOLCHAIN_FILE=conan_toolchain.cmake '
-                           .format(vcvars))
-        client.run_command("{} && cmake --build .".format(vcvars))
-        client.run_command("App")
-        self.assertIn("main: Release!", client.out)
-        self.assertIn("main _M_X64 defined", client.out)
```

Comment:
You are removing very useful and valid checks: the ``cppstd`` or the real compiler version. The other tests are using and checking it, I would prefer to keep this.